altair_saver/savers/_html.py

```diff
@@ -122,6 +122,7 @@ def __init__(
         spec: JSONDict,
         mode: Optional[str] = None,
         embed_options: Optional[JSONDict] = None,
+        vega_cli_options: Optional[List] = None,
```

Comment:
Strange that the vega_cli_options are passed here, when they're not used.

Reply:
The issue is that we need to pass `vega_cli_options` into all of the `Saver` methods. We can tweak that by instead updating `kwargs` manually:

```python3
if embed_options is None:
    embed_options = alt.renderers.options.get("embed_options", None)
if vega_cli_options is None:
    vega_cli_options = alt.renderers.options.get(
        "vega_cli_options", None
    )
kwargs["vega_cli_options"] = vega_cli_options

Saver = _select_saver(method, mode=mode, fmt=fmt, fp=fp)
saver = Saver(
    spec,
    mode=mode,
    embed_options=embed_options,
    **kwargs,
)
```

Preference?

Reply:
Note the above needs to check `issubclass(Saver, NodeSaver)` (or for equality). Updated as such in the 3rd commit on this PR.

Reply:
We're now just passing these through as `kwargs`. This does mean each `save()` invocation requires passing `vega_cli_optoins` if being called explicitly.